zebra-consensus/src/script.rs

```diff
@@ -33,16 +18,12 @@ const UTXO_LOOKUP_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(
 /// The asynchronous script verification design is documented in [RFC4].
 ///
 /// [RFC4]: https://zebra.zfnd.org/dev/rfcs/0004-asynchronous-script-verification.html
-#[derive(Debug, Clone)]
-pub struct Verifier<ZS> {
-    state: Timeout<ZS>,
-}
+#[derive(Debug, Clone, Default)]
+pub struct Verifier {}
```

Comment:
Nit: we can make this a unit struct:

```suggestion
pub struct Verifier;
```

Reply:
Done in b36577315 / #3446. I completely forgot this syntax :sweat_smile: